packages/editor/src/components/widgets/table/cell/ExpandableCell.tsx

```diff
@@ -39,12 +40,18 @@ export function ExpandableCell<TData>({ cell, isLoaded, loadChildren, isUnknown,
       ) : isUnknown === true ? (
         '⛔'
       ) : (
+        <>{icon && <IvyIcon icon={icon} />}</>
+      )}{' '}
+      {helperInfo ? (
         <>
-          🔵
-          {icon && <IvyIcon icon={icon} />}
+          <span className='row-expand-label'>
+            <b>{cell.getValue() as string}</b>
+          </span>
+          <span className='row-expand-label'>- {helperInfo}</span>
         </>
-      )}{' '}
-      <span className='row-expand-label'>{cell.getValue() as string}</span>
+      ) : (
+        <span className='row-expand-label'>{cell.getValue() as string}</span>
+      )}
```

Comment:
instead of this big conditional stuff move the additional info more to the place where it is needed, so you can reduce duplication!
And e.g. use css to make the difference between normal text and addional info

```jsx
<span className='row-expand-label'>{cell.getValue() as string}</span>
{helperInfo ?? <span className='row-expand-label-info'>- {helperInfo}</span>}
```